src/components/common/SanityReadmore/__tests__/SanityReadmore.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { describe, it } from 'vitest'
+
+import { SanityReadmore } from '@/components/common/SanityReadmore'
+import { mockErrorResponse } from '@/mocks/server'
+import { render, screen } from '@/test-utils'
+
+describe('SanityReadmore', () => {
+  describe('Gitt at Sanity er aktivert og innhold finnes', () => {
+    it('rendrer ReadMore med korrekte props og innhold', async () => {
+      render(
+        <SanityReadmore id="hva_er_opphold_utenfor_norge">
+          <div>Child content</div>
+        </SanityReadmore>
+      )
+
+      const readMoreElement = await screen.findByTestId(
+        'hva_er_opphold_utenfor_norge'
+      )
+      expect(readMoreElement).toBeVisible()
+
+      expect(screen.getByText('Hva som er opphold utenfor Norge')).toBeVisible()
+    })
+  })
+
+  describe('Gitt at Sanity er deaktivert', () => {
+    it('viser fallback innhold og ikke Sanity innhold', async () => {
+      mockErrorResponse('/feature/pensjonskalkulator.hent-tekster-fra-sanity')
+
+      render(
+        <SanityReadmore id="hva_er_opphold_utenfor_norge">
+          <p>Fallback innhold</p>
+        </SanityReadmore>
+      )
+
+      const readMoreElement = await screen.getByText('Fallback innhold')
+      expect(readMoreElement).toBeVisible()
+      readMoreElement.click()
```

Comment:
Tror ikke det har noen hensikt å klikke på fallback-innholdet når det bare er en `<p>`?
Du trenger forresten ikke `await` her siden vi ikke trenger å vente på at elementet skal dukke opp.
```suggestion
      expect(screen.getByText('Fallback innhold')).toBeVisible()
```